### ISSUE TITLE
The result is all black

### ISSUE THREAD

**Issue by qiaogh97** [matheusgadelha/PrGAN#4]
Hello @matheusgadelha !
I run this PrGAN.py code and have some questions. After training 25 epochs, my result is all black and the 'volume*.npy' is all about zero.
My tensorflow's version is 1.3.0, so I modify some code to fit my version. The biggest change is that I use tf.layers.batch_normalization() instead of ops.BatchNormalization(), because this code can't run correctly.
And I use 'python src/PrGAN.py -d airplane64 --train -e 25 -ims 64' as the airplane64's size is 64 but default size is 32.
The code in PrGAN.py line 227 is interesting. I think the channel should be 'base_filters/8', in your code this is '1'.
Thank you for your reply!
